src/styles/tools/_functions.scss

```diff
@@ -15,3 +15,13 @@
 @function calculate_unitless_line_height($line_height, $font_size) {
   @return $line_height / $font_size;
 }
+
+// Pixels to ems.
+@function px-to-em($px) {
+  @return $px / $base-font-size * 1em;
+}
+
+// Pixels to rems.
```

Comment:
Do you think this is worth bringing across? Can we just keep using px instead?

Reply:
I'm using it in the toc widget and we use it a lot in d.c.c.'s css so it seemed like it's worth pulling across.